src/drom_lib/subst.ml

```diff
@@ -38,7 +38,7 @@ let project_brace (_, p) v =
   | "min-edition" -> p.min_edition
   | "github-organization" -> maybe_string p.github_organization
   | "authors-as-strings" ->
-    String.concat " " (List.map (Printf.sprintf "%S") p.authors)
+    String.concat ", " (List.map (Printf.sprintf "%S") p.authors)
```

Comment:
I believe the previous behavior was a bug.

Reply:
I think the problem is more about the name: there should probably be 2 variables `authors-with-spaces` and `authors-with-commas`. But indeed, it is only used in a `toml` file, so it should use commas.